litestar/dto/dataclass_dto_factory.py

```diff
@@ -47,7 +45,7 @@ def generate_field_definitions(
                     field_definition=field_definition,
                     default_factory=default_factory,
                     dto_field=dc_field.metadata.get(DTO_FIELD_META_KEY, DTOField()),
-                    unique_model_name=get_fully_qualified_class_name(model_type),
+                    model_name=model_type.__name__,
```

Comment:
Should we use `__qualname__` here instead?

Reply:
The unique naming was brought in to prevent errors with openapi schema generation caused by having multiple transfer models with the same name. Has that been addressed some other way?

Reply:
> Has that been addressed some other way?

It has not, no. As far as I’m aware, the behaviour is still the same?

Reply:
it has, yes. and unlike the previous version which created some weird random names with numbers and made the open api docs half usable, this is actually readable.